sql/core/src/main/scala/org/apache/spark/sql/execution/BaseScriptTransformationExec.scala

```diff
@@ -17,7 +17,7 @@
 
 package org.apache.spark.sql.execution
 
-import java.io._
+import java.io.{BufferedReader, File, InputStream, InputStreamReader, OutputStream}
```

Comment:
Oh, I don't know why I thought it's more than 5+. It's more than 6+ (https://github.com/databricks/scala-style-guide#imports).

Reply:
> Oh, I don't know why I thought it's more than 5+. It's more than 6+ (https://github.com/databricks/scala-style-guide#imports).

Seems my local ideal's rule is 5, emmm I need to check this.